install/kubernetes/cilium/values.yaml

```diff
@@ -1331,6 +1331,12 @@ hubble:
       # --- The port to use when the service type is set to NodePort.
       nodePort: 31235
 
+
+    # -- Defines base url prefix for all hubble-ui http requests.
+    # It needs to be changed in case if ingress for hubble-ui is configured under some sub-path.
+    # Trailing `/` is required for custom path, ex. `/service-map/`
+    baseUrl: "/"
+
```

Comment:
[helm-charts CI is complaining](https://github.com/cilium/cilium/actions/runs/4123580601/jobs/7121771474) because this change isn't in `values.yaml.tmpl`:
> error: cilium/values.yaml has been modified
Make sure to apply your changes to cilium/values.yaml.tmpl and run 'make -C install/kubernetes cilium/values.yaml' to regenerate.